src/pages/organize/[orgId]/index.tsx

```diff
@@ -51,68 +49,67 @@ type OrganizePageProps = {
-                        <View borderColor="gray-400" borderWidth="thick" margin="size-200" padding="size-200">
-                            <DashboardCampaigns orgId={ orgId }/>
-                        </View>
-                        <View borderColor="gray-400" borderWidth="thick" margin="size-200" padding="size-200">
-                            <DashboardPeople orgId={ orgId }/>
-                        </View>
-                        <View borderColor="gray-400" borderWidth="thick" margin="size-200">
-                            Areas
-                        </View>
-                    </View>
-                    <View borderColor="gray-400" borderWidth="thick" height="size-1200" margin="size-200" width="50%">Inbox</View>
-                </Flex>
-            </View>
+            <Box className={ classes.responsiveFlexBox } p={ 3 }>
+                <Box p={ 2 }>
+                    <Avatar src={ apiUrl(`/orgs/${orgId}/avatar`) } style={{ height: '100px', width: '100px' }}></Avatar>
+                </Box>
+                <Box display="flex" flexDirection="column" p={ 2 }>
+                    <Typography component="h1" variant="h3">
+                        { orgQuery.data?.title }
+                    </Typography>
+                    <Typography color="primary">
+                        <Box className={ classes.responsiveFlexBox } p={ 0 }>
+                            <Box display="flex" p={ 1 } pl={ 0 }>
+                                <Public />
+                                <NextLink href={ `/o/${orgId}` } passHref>
+                                    <Link color="inherit">
+                                        <Msg id="pages.organize.linkGroup.public"/>
+                                    </Link>
+                                </NextLink>
+                            </Box>
+                            <Box display="flex" p={ 1 }>
+                                <SupervisorAccount />
+                                <NextLink href={ `/organize/${orgId}/organization` } passHref>
+                                    <Link color="inherit">
+                                        <Msg id="pages.organize.linkGroup.manage"/>
+                                    </Link>
+                                </NextLink>
+                            </Box>
+                            <Box display="flex" p={ 1 }>
+                                <Settings />
+                                <NextLink href={ `/organize/${orgId}/settings` } passHref>
+                                    <Link color="inherit">
+                                        <Msg id="pages.organize.linkGroup.settings"/>
+                                    </Link>
+                                </NextLink>
+                            </Box>
+                        </Box>
+                    </Typography>
+                </Box>
+            </Box>
+            <Box className={ classes.responsiveFlexBox } p={ 2 }>
```

Comment:
Checkout  https://material-ui.com/components/grid/#grid-with-breakpoints for this

Reply:
Decided to use Grid in the next page, I will make an issue for the minor changes we discused so the branch can be merged ASAP